docs/learn/learn-staking.md

```diff
@@ -221,89 +396,89 @@ The following example should clarify the above. For simplicity, we have the foll
 
-We designed this hierarchy of separate key types so that validator operators and nominators can
-protect themselves much better than in systems with only one key. As a rule, you lose security
-anytime you use one key for multiple roles, or even if you use keys related by derivation. You
-should never use any account key for a "hot" session key in particular.
+:::info Payouts
 
-Controller and Stash account keys can be either sr25519 or ed25519. For more on how keys are used in
-Polkadot and the cryptography behind it [see here](learn-keys.md).
+Payouts are unclaimed rewards waiting to be paid out to both validators and nominators. If you go to the Staking payouts page on
+[Polkadot-JS](https://polkadot.js.org/apps/#/staking), you will see a list of all validators
+that you have nominated in the past 84 eras and for which you have not yet received a payout. The payout page is visible only to stakers.
 
-## Validators and Nominators
+:::
 
-Since validator slots are limited, most of those who wish to stake their DOT and contribute economic
-security to the network will be nominators.
+Each validator as well as their nominators have the option to trigger the payout for all unclaimed eras. Note that this will pay everyone
+who was nominating that validator during those eras. Therefore, you may not
+see anything in this tab, yet still have received a payout if somebody (generally, but not
+necessarily, another nominator or the validator operator) has triggered the payout for that
+validator for that era.
 
-Validators do most of the heavy lifting: they produce new block candidates in BABE, vote and come to
-consensus in GRANDPA, validate the state transition function of parachains, and possibly some other
-responsibilities regarding data availability and [XCM](learn-cross-consensus.md).
+:::warning Time limit
 
-Nominators, on the other hand, have far fewer responsibilities. Those include monitoring their
-validators' performance (uptime), keeping an eye on changing commission rates (a validator can
-change commission at any time), and general health monitoring of their and their validators'
-account. Thus, while not set-it-and-forget-it, a nominator's experience is relatively hands-off
-compared to a validator's.
+If nobody claims your staking rewards within 84 eras, then you will not be able to claim them and they will be lost. Additionally, if the validator unbonds all their own stake, any pending payouts will also be lost. Since unbonding takes {{ polkadot: <RPC network="polkadot" path="query.staking.bondingDuration" defaultValue={28}/> days :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.bondingDuration" defaultValue={7}/> days :kusama }}, nominators should check if they have pending payouts at least this often.
```

Comment:
```suggestion
If nobody claims your staking rewards within 84 eras, you will not be able to claim them and they will be lost. Additionally, if the validator unbonds all of their self stake, any pending payouts will also be lost. Since unbonding takes {{ polkadot: 28 days :polkadot }}{{ kusama: 7 days :kusama }}, nominators should periodically check if they have pending payouts.
```